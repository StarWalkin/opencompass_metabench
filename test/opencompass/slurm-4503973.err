  0%|          | 0/2 [00:00<?, ?it/s]                                       0%|          | 0/2 [00:00<?, ?it/s]slurmstepd: error: *** JOB 4503973 ON gpua052 CANCELLED AT 2024-08-17T19:07:59 ***
