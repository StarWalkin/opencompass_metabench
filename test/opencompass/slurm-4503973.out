/scratch/bcwg/Metabench/anaconda/envs/opencompass/bin/python3
08/17 18:59:47 - OpenCompass - INFO - Loading gsm8k_gen: configs/datasets/gsm8k/gsm8k_gen.py
08/17 18:59:47 - OpenCompass - INFO - Loading hf_gemma2_9b_it: configs/models/gemma/hf_gemma2_9b_it.py
08/17 18:59:47 - OpenCompass - INFO - Loading hf_phi_3_small_8k_instruct: configs/models/phi/hf_phi_3_small_8k_instruct.py
08/17 18:59:47 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/17 18:59:48 - OpenCompass - INFO - Current exp folder: outputs/default/20240817_185948
08/17 18:59:48 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/17 18:59:48 - OpenCompass - INFO - Partitioned into 2 tasks.
